spec/features/admin/chapters_spec.rb

```diff
@@ -11,13 +11,13 @@
     scenario "an admin can create a new chapter" do
       visit new_admin_chapter_path
 
-      fill_in "Name", with: "Codebar Brighton"
+      fill_in "Name", with: "codebar Brighton"
       fill_in "Email", with: "brighton@codebar.io"
       fill_in "City", with: "Brighton"
 
       click_on "Create chapter"
 
-      expect(page).to have_content("Chapter Codebar Brighton has been succesfuly created")
+      expect(page).to have_content("Chapter codebar Brighton has been succesfuly created")
```

Comment:
Should the message simply be `"codebar Brighton has been successfully created"`, without the `"Chapter"` part?